internal/polar/client.go

```
@@ -0,0 +1,558 @@
+	Activation       struct {
+		Id           string         `json:"id"`
+		LicenseKeyId string         `json:"license_key_id"`
+		Label        string         `json:"label"`
+		Meta         map[string]any `json:"meta"`
+		CreatedAt    time.Time      `json:"created_at"`
+		ModifiedAt   any            `json:"modified_at"`
+	} `json:"activation"`
+}
+
+func (v *ValidateResp) ValidLicense() bool {
+	if v.Status == "granted" {
```

golangci-lint (2.12.2)

[error] S1008: should use 'return v.Status == "granted"' instead of 'if v.Status == "granted" { return true }; return false'

(staticcheck)